src/heads-up/components/app-tooltip.vue

```diff
@@ -1,129 +1,168 @@
 <template>
-  <span v-tooltip="options">
+  <div
+    v-tooltip.top="{ ref: 'tooltip' }"
+  >
     <slot />
+    <slot name="value" />
 
-    <span
-      ref="info"
-      class="visually-hidden"
-    >
+    <span ref="tooltip">
       <slot name="info" />
+      <slot name="link" />
     </span>
-  </span>
+  </div>
 </template>
 
 <script>
 export default {
-  data() {
-    return {
-      options: {
-        content: {}
-      }
-    }
-  },
-  mounted() {
-    this.options.content = `${ this.$refs.info.innerHTML }`
-  }
 }
 </script>
 
 <style>
-.tooltip {
-  display: block !important;
-  z-index: 10000;
-}
+  .vue-tooltip {
+    width: 150px;
+    padding: 5px 8px;
+    background-color: var(--selection-fg-color);
+    border-radius: 2px;
+    color: hsl(0, 0%, 20%);
+    line-height: 14px;
+    filter: drop-shadow(0 1px 2px hsla(0, 0%, 0%, 0.3));
+    border: 1px solid hsla(0, 0%, 0%, 0.1);
+  }
```

Comment:
Maybe you could add a hit area instead of using the delay? Something like this: 

```
.vue-tooltip:after {
  content: "";
  position: absolute;
  top: 100%;
  left: 0;
  width: 100%;
  height: 1.5rem;
}
```